src/main/java/frc/robot/RobotContainer.java

```diff
@@ -23,10 +23,12 @@
 public class RobotContainer {
   //init controller
   XboxController xbox = new XboxController(Constants.controllerport);
+  
   // The robot's subsystems and commands are defined here...
   driveTrain m_driveTrain;
   output m_output; //output
 
+  input m_input;
```

Comment:
idk if a comment is necessary here but if it is (Jacob?), briefly state what that variable is.

Reply:
When in doubt add comments describing what this variable contains and does.